node_agent/main.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"github.com/golang/glog"
+	"istio.io/auth/node_agent/na"
+)
+
+var (
+	naConfig na.Config
+	env      *string
+)
+
+func init() {
+	naConfig.ServiceIdentity = flag.String("service-identity", "",
+		"Service Identity the node agent is managing")
+	naConfig.ServiceIdentityOrg = flag.String("org", "Juju org", "organization for the cert")
```

Comment:
@lookuptable @wattli Why do we have this field?
https://github.com/istio/auth/blob/eae885867672af349a9a59c9bf0531e7f13d9160/certmanager/generate_cert.go#L174
Is this intentionally serving a purpose or due to other reasons?